resources/metadata/kubernetesDeploy.yaml

```diff
@@ -550,17 +550,7 @@ spec:
           params:
             - name: deployTool
               value: helm3
-    - image: dtzar/helm-kubectl:2.17.0
-      workingDir: /config
-      options:
-        - name: -u
-          value: "0"
-      conditions:
-        - conditionRef: strings-equal
-          params:
-            - name: deployTool
-              value: helm
-    - image: dtzar/helm-kubectl:2.17.0
```

Comment:
I suppose what part was removed accidentally? If so, please fix and don't forget to regenerate the code.